modules/http/src/test/java/org/mule/module/http/functional/listener/HttpListenerPersistentConnectionsTestCase1_0.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.http.functional.listener;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mule.module.http.api.HttpHeaders.Values.CLOSE;
+import static org.mule.module.http.api.HttpHeaders.Values.KEEP_ALIVE;
+
+import java.io.IOException;
+
+import org.apache.http.HttpVersion;
+import org.junit.Test;
+
+public class HttpListenerPersistentConnectionsTestCase1_0 extends HttpListenerPersistentConnectionsTestCase
```

Comment:
Don't use an underscore in the class name, and put the protocol version before "TestCase".